docs/content/Getting-Started/user-guide.md

```diff
@@ -1,11 +1,104 @@
+
+The following picture shows a consumer's view of the architecture of the current state of development of KubeStellar MCCM.
+
+![Consumer's view of KubeStellar MCCM](KubeStellar-Usage.svg "Consumer's view of KubeStellar MCCM")
+
+KubeStellar MCCM is currently set in the context of [kcp](https://github.com/kcp-dev/kcp) --- release v0.11.0, to be specific. KubeStellar MCCM makes use of the multiplicity of kcp workspaces to separate various concerns: separate clients from each other and from implementation, allow a client to separate workloads that might clash in cluster-scoped objects, allow a client to have an inventory that does not comfortably fit in one etcd cluster (as you have seen, high scale is a concern in the interface but not yet in the implementation).
+
+A deployment of KubeStellar MCCM supports multiple clients. Each owns some Workload Execution Clusters (WECs), which are Kubernetes clusters of whatever sort the client wants. A client wants to manage the configuration of the Kubernetes API objects in the WECs from a central place. A client has an inventory of WECs, and manages one or more inventory spaces (ISes) to hold that inventory. The inventory holds a pair of kube-style API objects per WEC, of kind `SyncTarget` and `Location`. A client has one or more workloads that the client wants to be run on the WECs. KubeStellar MCCM does not have an opinion on how a client divides its objects between "workloads"; the MCCM interfaces deal directly with kube-style API objects and we use the term "workload" only informally in documentation. A client maintains its workload descriptions in Workload Description Spaces (WDSes). A client maintains just one copy of each workload object in the core (the MCCM core is the central place where the clients interact with MCCM; it includes the ISes and WDSes). To convey its wishes about what goes where, a client maintains control objects of kind `EdgePlacement` in the WDSes. Each `EdgePlacement` object binds a selection of WECs with both (a) a definition of which workload objects are downsynced and (b) a definition of which workload objects are upsynced. The MCCM implementation implements the prescribed behavior.
+
+In KubeStellar MCCM there are downsynced objects and upsynced objects. A downsynced object is one that a client creates in a WDS. Its desired state propagates from WDS to WEC, and its reported state propagates back towards the core. An upsynced object is created in a WEC and MCCM transports it back towards the core.
+
+For each WEC there is a setup step that establishes its relationship with the core. This is called "WEC bootstrapping".
+
+KubeStellar MCCM uses the APIExport/APIBinding technology of kcp to separate consumers and the provider of MCCM functionality and the uses and definition of the Kubernetes resources for the management of containerized workloads.
```

Comment:
Does this needs to be in the user guide?  Or is this better contained in a design document?

Reply:
By "this" you mean both of the paragraphs you highlighted?  I assume so.

I think the paragraph about bootstrapping is needed because this is very visible to users.

I included the paragraph about kcp APIExport/APIBinding also because it is visible to users.  Their WDS needs APIBindings to the MCCM exports, and needs APIBindings to the exports of the relevant workload resources.